coordinator/src/main/java/org/apache/uniffle/coordinator/web/servlet/NodesServlet.java

```diff
@@ -37,10 +38,17 @@ public NodesServlet(CoordinatorServer coordinator) {
   }
 
   @Override
-  protected Response<List<ServerNode>> handleGet(HttpServletRequest req, HttpServletResponse resp) {
-    List<ServerNode> serverList = coordinator.getClusterManager().getServerList(Collections.EMPTY_SET);
+  protected Response handleGet(HttpServletRequest req, HttpServletResponse resp) {
+    List<ServerNode> serverList;
     String id = req.getParameter("id");
     String status = req.getParameter("status");
+    if (status != null && status.equalsIgnoreCase(ServerStatus.UNHEALTHY.name())) {
```

Comment:
```suggestion
    if (ServerStatus.UNHEALTHY.name().equalsIgnoreCase(status)) {
```

Reply:
If the status parameter is not passed in the URL, status.equalsignorecase will raise a null pointer exception, but we are allowed not to pass the status parameter when the active state is present

Reply:
You can see the implement of `equalsIgnoreCase`. The method can handle null parameter.
```
    public boolean equalsIgnoreCase(String anotherString) {
        return (this == anotherString) ? true
                : (anotherString != null)
                && (anotherString.value.length == value.length)
                && regionMatches(true, 0, anotherString, 0, value.length);
    }
```